### DISCUSSION TITLE
Any tips for new Shadcn-Svelte user?

### DISCUSSION THREAD

**Discussion by emmabyte** [huntabyte/shadcn-svelte#1054] [General]
Hey everyone, I'm migrating from Skeleton to shadcn-svelte and was wondering if anyone had any tips or things they wish they knew when they just started using shadcn-svelte. Thanks!

**Comment by shyakadavis** [comment]
Hello, @shawnlong636 

The only tip I would have for you is; `don't override the variable names`. This will lead to more work for you.

Wrote something small on my experience from a recent project here: https://shyakadavis.me/projects/redesign/steam#lessons

Obviously, there are more tips one could offer, but I always forget to document them. But as soon as I remember, I'll let you know. 🙂

Be safe.

**Reply by emmabyte** [reply]
Appreciate it man, thanks!
